<launch>
  <arg name="mav_name" default="cygnus" />
  <arg name="namespace" default="$(arg mav_name)"/>
  <group ns="$(arg namespace)">
   <node name="mav_control" pkg="cygnus_control" type="control_posicion_modular" respawn="true" clear_params="true" output="screen">
      <remap from="odometry" to="ground_truth/odometry" />
      <param name="verbose" value="true" />
      <param name="use_rc_teleop" value="false" />
      <param name="reference_frame" value="world" />
    </node>  

    <node name="PID_attitude_controller" pkg="cygnus_control" type="mav_pid_attitude_controller_node" respawn="true" clear_params="true" output="screen">
      <remap from="odometry" to="ground_truth/odometry" />
      <rosparam file="$(find cygnus_control)/resources/PID_attitude_$(arg mav_name).yaml" />
    </node> 

  <!-- <node name="pose_sensor" pkg="msf_updates" type="pose_sensor" clear_params="true" output="screen">
     <remap from="msf_updates/transform_input" to="ground_truth/transform" />
     <remap from="msf_core/imu_state_input" to="imu" />
     <rosparam file="$(find mav_linear_mpc)/resources/msf_parameters_sim.yaml"/>
  </node> -->

  </group>
</launch>
